iFixit/react-commerce
# Refactor product list ancestry code

closes #1837 

## QA

1. Open a [product list preview](https://react-commerce-git-refactor-product-list-ancestors-ifixit.vercel.app/Parts/Android_Phone)
2. Test that product list breadcrumbs work and verify that they should include the store home page breadcrumb
